third_party/move/move-compiler-v2/legacy-move-compiler/src/shared/mod.rs

```diff
@@ -705,6 +706,21 @@ pub mod known_attributes {
         ModuleLock,
     }
 
+    #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+    pub enum StructAPIAttribute {
+        /// Marks a struct as having API for packing and unpacking
```

Comment:
If I've understood the code correctly, I don't think these are "marking the struct" as having certain APIs, these are "marking a function as being the API for ..".